capa/features/extractors/dnfile/insn.py

```diff
@@ -97,17 +98,32 @@ def extract_insn_api_features(fh: FunctionHandle, bh, ih: InsnHandle) -> Iterato
     if insn.opcode not in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli, OpCodes.Newobj):
         return
 
-    callee: Union[DnType, DnUnmanagedMethod, None] = get_callee(fh.ctx, insn.operand.value)
+    api_token: int
+
+    # ECMA: MethodSpec table records the signature of an instantiated generic method
+    #   Method (an index into the MethodDef or MemberRef table, specifying to which generic method this row refers)
+    #   Instantiation(an index into the Blob heap holding the signature of this instantiation)
+    if is_mdtable(insn.operand.table, "GenericMethod"):
```

Comment:
is there really no other way to tell how to interpret this union? are they guaranteed to never overlap?

this is more commentary/double checking than a request for changes.